### PR TITLE
Healthcare 13159 review page update

### PR DESCRIPTION
**Pull request by mdewey**
## Description
Formated the review page for the questionnaire app. Updated the Demographics section to look more consistent with the forms system styling. 

One part, the questions list,  is still outstanding and waiting on the Accessibility Team. 

## Testing done
- [x] unit tests 

## Screenshots
screencapture-localhost-3001-healthcare-questionnaire-review-and-submit-2020-10-05-09_29_25

## Acceptance criteria
- [x] Matches the [mockups](https://app.zeplin.io/project/5f3d86a458ea2231e60a4780/screen/5f57bbe3f8ccf83e7b743b43)

## Definition of done
- [x] [Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13159)

### CODE REVIEW

File: src/applications/healthcare/questionnaire/sass/review.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
+@import "../../../../platform/forms/sass/m-form-confirmation";
+
+
+
+.healthcare-questionnaire-review{
+  div.top-row.review-row{
+    border-top: none;
+  }
+}
```
Comment (dillo) [inline src/applications/healthcare/questionnaire/sass/review.scss:16]: Do we care about adding a new line to the end of certain files?

File: src/applications/healthcare/questionnaire/components/get-help/ReviewFooter.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+              </li>
+              <li>
+                Start a{' '}
+                <a href="https://www.veteranscrisisline.net/ChatTermsOfService.aspx?account=Veterans%20Chat/">
+                  confidential Veteran chat
+                </a>{' '}
+                , or
+              </li>
+              <li>
+                Text <a href="sms:838255">838255</a>.
```
Comment (dillo) [inline src/applications/healthcare/questionnaire/components/get-help/ReviewFooter.jsx:37]: Should numbers like these be assigned to some constant?

File: src/applications/healthcare/questionnaire/components/veteran-info/utils/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import { states50AndDC } from 'vets-json-schema/dist/constants.json';
+
+const stateCodeToFullState = stateCode => {
+  return states50AndDC.filter(f => f.value === stateCode)[0]?.label;
+};
+
+const addressToDisplay = address => {
+  const rv = [];
```
Comment (dillo) [inline src/applications/healthcare/questionnaire/components/veteran-info/utils/index.js:8]: Could we come up with a slightly more descriptive const than `rv`? Not sure about the JS convention here so I'm going to default to your expertise!

File: src/applications/healthcare/questionnaire/sass/review.scss (hunk 1 above)
Comment (mdewey) [reply src/applications/healthcare/questionnaire/sass/review.scss:16]: Our linter isn't set up for SCSS files yet, so there is no standard. I'll make it consistent for our files.

File: src/applications/healthcare/questionnaire/components/get-help/ReviewFooter.jsx (hunk 2 above)
Comment (mdewey) [reply src/applications/healthcare/questionnaire/components/get-help/ReviewFooter.jsx:37]: Probably. It's in the codebase in a few(7) other places  and no where is it a constant. To make it a global constant is another matter, but I will pull this out. at least locally.

File: src/applications/healthcare/questionnaire/components/veteran-info/utils/index.js (hunk 3 above)
Comment (mdewey) [reply src/applications/healthcare/questionnaire/components/veteran-info/utils/index.js:8]: I like `rv`, but I get the read-ability angle. I will update

Comment (dillo) [reply src/applications/healthcare/questionnaire/components/veteran-info/utils/index.js:8]: That really was a nitpick. Sorry :)

**Review by dillo — APPROVED** [review]

**Pull request merged**